pkg/webhook/internal/certprovider/doc.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
This probably shouldn't be internal

Reply:
I'd like to move it out when both the `CertProvider` and the WH interface are ready.